apps-script/README.md

```diff
@@ -0,0 +1,13 @@
+# Apps Script
+
+This is the code to populate the repos tab on the [dependencies spreadsheet](https://docs.google.com/spreadsheets/d/137KZhjctJ8qTKYPnq2QNVkyoIC6ok7KA2G1vErNC6Oo/edit#gid=1160366178), it should be kept in sync with the code in the Apps Script.
+
+## How to make changes
+
+After getting your changes approved and merged, you need to follow these steps:
+
+- Go to https://docs.google.com/spreadsheets/d/137KZhjctJ8qTKYPnq2QNVkyoIC6ok7KA2G1vErNC6Oo/edit#gid=1160366178
```

Comment:
Suggest we avoid repeating the link, to avoid having to update it in two places.
```suggestion
- Go to the spreadsheet
```